.github/workflows/ci_cd_pr.yml

```
@@ -108,7 +110,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: changelog-fragment
     steps:
-      - uses: ansys/actions/doc-style@main
+      - uses: ansys/actions/doc-style@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-style@main
```
Needs to be reverted before merging